### PR TITLE
Update admin records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1911

Much of the updates needed to the admin records in Qatar was done though https://github.com/whosonfirst-data/whosonfirst-data/pull/1523. This PR reviews that work, updating two dozen records with more detailed geometries.

This PR:

- Updates and adds neighbourhood records along the coastal area of Doha
- Adds four new locality polygons for locality records along the cast of Qatar, south of Doha
- Includes some superseding work / property swapping in commit 5e093e7
- Updated the locality of Ar Rayyan, demoting locality points around this area (and Doha) to new neighbourhood records

This PR will need to be run across PIP tools to update hierarchies.

### CODE REVIEW

**Comment by stepps00** [comment]
After reviewing these changes, I'll need to update the area of Al-Rayyan with a new locality polygon and fix some of the existing Point records in that same area. I'll remove the review request, commit those changes, then re-request a review.

**Comment by stepps00** [comment]
This PR now includes updated locality polygons for Ar Rayyan, as well as demoting some locality points to the neighbourhood placetype.

File: data/421/190/363/421190363.geojson [hunk 1]
```diff
@@ -646,7 +646,7 @@
     "wof:geom_alt":[
         "quattroshapes"
     ],
-    "wof:geomhash":"5800acfadc5639d4988594b67e75b50c",
+    "wof:geomhash":"fee59cd300f0a2d92a31675a5fdd7623",
     "wof:hierarchy":[
```
Comment (nvkelso) [inline data/421/190/363/421190363.geojson:650]: Noting Doha needs a dual hierarchy because it crosses admin boundaries now with this change.

**Review by nvkelso — APPROVED** [review]
LGTM

**Comment by stepps00** [comment]
This PR should be ready to merge, the most recent commit includes PIP work for all updated records.

I'll QA a bit more then merge this work.

File: data/421/190/363/421190363.geojson (hunk 1 above)
Comment (stepps00) [reply data/421/190/363/421190363.geojson:659]: addressed in f7d63e7

**Comment by stepps00** [comment]
I've added an additional commit to update the admin records around Doha's Airport. This work includes:

- Flagging the airport `campus` and `neighbourhood` records as coterminous
- Giving the `campus` record the same geometry as the neighbourhood record
- Demoting the existing `campus` point to an alt geometry
- Fixing superseding issues with three airport records

**Comment by stepps00** [comment]
This PR looks good, all changes validate.

**Pull request merged**